src/activity-log/index.html

```diff
@@ -52,9 +52,9 @@
                     </td>
                     <td class="u-w65pct">
                         <a href="{{ item.permalink }}">
-                            <h4 class="u-mb0">
+                            <span class="h4 u-mb0">
```

Comment:
Admittedly, I don't know what the accessibility implications of that technique are.